network_simulator/flow.py

```diff
@@ -89,5 +92,218 @@ def handle_packet_success(self, packet_id):
     def __repr__(self):
         return str(self.__dict__)
 
-# TODO(team): Flow-related Event subclasses
-# TODO(team): Flow subclasses for at least two TCP algorithms.
+
+
+class InitiateFlowEvent(Event):
```

Comment:
This is called during the initializer code, and that code does not know what "packet_ids" is.

What needs to happen is that, during run(), you need to figure out the first set of packets to send for the Flow. These should be {0, ..., N} where N is determined by the window size of the flow (defaults to 1 during construction) and the number of packets left to send.